sdk/cosmos/azure-cosmos-encryption/src/test/java/com/azure/cosmos/encryption/EncryptionAsyncApiCrudTest.java

```diff
@@ -154,6 +157,52 @@ public void queryItems() {
         }
     }
 
+    @Test(groups = {"encryption"}, timeOut = TIMEOUT)
+    public void queryItemsAggregate() {
+        long startTime = Instant.now().getEpochSecond();
+        List<String> actualIds = new ArrayList<>();
+        EncryptionPojo properties = getItem(UUID.randomUUID().toString());
+        cosmosEncryptionAsyncContainer.createItem(properties, new PartitionKey(properties.getMypk()),
+            new CosmosItemRequestOptions()).block();
+        actualIds.add(properties.getId());
+        properties = getItem(UUID.randomUUID().toString());
+        cosmosEncryptionAsyncContainer.createItem(properties, new PartitionKey(properties.getMypk()),
+            new CosmosItemRequestOptions()).block();
+        actualIds.add(properties.getId());
+        properties = getItem(UUID.randomUUID().toString());
+        cosmosEncryptionAsyncContainer.createItem(properties, new PartitionKey(properties.getMypk()),
+            new CosmosItemRequestOptions()).block();
+        actualIds.add(properties.getId());
+
+        // MAX query
+        String query1 = String.format("Select value max(c._ts) from c");
+        CosmosQueryRequestOptions cosmosQueryRequestOptions1 = new CosmosQueryRequestOptions();
+
+        SqlQuerySpec querySpec1 = new SqlQuerySpec(query1);
+        CosmosPagedFlux<EncryptionPojo> feedResponseIterator1 =
+            cosmosEncryptionAsyncContainer.queryItems(querySpec1, cosmosQueryRequestOptions1, EncryptionPojo.class);
+        List<EncryptionPojo> feedResponse1 = feedResponseIterator1.byPage().blockFirst().getResults();
+        long endTime = Instant.now().getEpochSecond();
+
+        int timeStamp = 0;
+        for (Object pojo: feedResponse1) {
+            timeStamp = Integer.parseInt(pojo.toString());
+        }
+        Assert.assertTrue(timeStamp > startTime);
+        Assert.assertTrue(timeStamp <= endTime);
```

Comment:
codestyle: please use assertJ assertion, not testng to be consistent with the rest of the tests.

Reply:
Removed, and added the correct assertion usage to be consistent with the rest of the test cases.